src/lib/ride.ts

```diff
@@ -37,3 +47,43 @@ export function unpauseRide(ride: RideModel) {
     });
   });
 }
+
+export type RideEvent = 'start' | 'stop' | 'tick';
```

Comment:
I tried this a lot of different ways. Writing a `ride_id` record to realtimedata table and watching that via watermelon, or watching the ride table via watermelon. But it ended up being pretty awkward to figure out if the ride had started stopped of was in progress and it's just a timer event. So felt like a event emitter was the best approach. I'll probably be useful running the upload tasks on ride stop too.

Reply:
it will also be useful for a status bar type widget that should be able to show if ride is active or paused, so I think its a good way to go